giskardpy/src/giskardpy/motion_statechart/motion_statechart.py

```diff
@@ -106,6 +106,82 @@ def __eq__(self, other: Self) -> bool:
         return str(self) == str(other)
 
 
+def _create_condition(
+    start_from: MotionStatechartNode,
+    condition_getter: Callable[[MotionStatechartNode], cas.Expression],
+    expected_value: cas.Expression,
+    combine_func: Callable[[cas.Expression, cas.Expression], cas.Expression],
+) -> cas.Expression:
```

Comment:
this method feels too complicated for me.
_create_condition is a very non descriptive name and a condition_getter as a callable attribute is difficult to understand too. i can see that you want to avoid that it depends on the specific condition. but then maybe this method should belong to TrinaryCondition?